dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/webapp/WebAppErrorReport.java

```diff
@@ -178,6 +176,11 @@ else if (rootCause instanceof UnavailableException) {
                 r.setErrorCode(HttpServletResponse.SC_SERVICE_UNAVAILABLE);
             }
         }
+        else if (rootCause instanceof IOException
+                        && th.getMessage() != null
+                        && th.getMessage().contains("CWWWC0005I")) {     //Servlet 6.0 added
```

Comment:
No way this can happen unless with Servlet 6 correct? No need to add a check?

Reply:
No, IOException has been thrown before.  This check ensure that a 400 is returned only if the IOException coming from the new path.  Without this check, a 400 won't be set.  or without the last check "th.getMessage().contains("CWWWC0005I", any IOException will get a 400 which may not be correct.